src/routes/+page.server.js

```diff
@@ -1,15 +1,20 @@
 import { getAllCollections } from '$utils/shopify';
+import { error } from '@sveltejs/kit';
 
-export async function get() {
+/** @type {import('./$types').RequestHandler} */
+export async function GET({ url }) {
   const res = await getAllCollections();
 
+  // okay this part is confusing
+  console.log(res);
+
   if (res.status === 200) {
     const products = res.body?.data?.collections?.edges;
 
     if (products) {
-      return {
-        body: { products }
-      };
+      console.log(products);
+
+      return { products };
     }
 
     return {
```

Comment:
this should be a `throw error(404)` or `throw error(404, 'Not found')`, and we probably need to handle the `status !== 200` case with a `throw error(res.status)` at the end